Modix.Bot/Behaviors/PromotionLoggingHandler.cs

```diff
@@ -137,7 +137,7 @@ private async Task<Embed> FormatPromotionNotification(long promotionActionId, Pr
             }
 
             return embed
-                .WithAuthor(await UserService.GetUserInformationAsync(targetCampaign.Subject.Id, data.GuildId))
+                .WithAuthor(await UserService.GetUserInformationAsync(data.GuildId, targetCampaign.Subject.Id))
```

Comment:
Should probably split into two lines of code like Cisien suggested for readability

Reply:
What is this? The linux repo? :^)

_Okay I'll fix it.._